pkg/assembler/backends/arangodb/backend.go

```diff
@@ -222,6 +222,12 @@ var mapEdgeToArangoEdgeCollection = map[model.Edge][]string{
 	model.EdgePackageHasSourceAt:               {hasMetadataPkgVersionEdgesStr, hasSourceAtPkgNameEdgesStr},
 	model.EdgePackageIsDependency:              {isDependencySubjectPkgEdgesStr},
 	model.EdgePackageIsOccurrence:              {isOccurrenceSubjectPkgEdgesStr},
+	model.EdgePackageNamePackageNamespace:      {},
```

Comment:
I know what you mean and I wondered what would have been the right value.
I put nothing because the query for getting the parent [1] has no `forOutBound` invocation so I thought no edge was required here.

[1] https://github.com/guacsec/guac/pull/1450/files#diff-3316b1f0bcbf22cc9d703a5a322d1e44c6467110b43f1393b44573d2a1ef36dbR1221

Reply:
hmm that's true. We can change it if we need it later but for now it seems good.